lib/pdf-editor-view.js

```diff
@@ -490,6 +495,32 @@ export default class PdfEditorView extends ScrollView {
     }, () => this.finishUpdate());
   }
 
+  computeMaxPageWidth(){
+    Promise.all(
+      _.range(1, this.pdfDocument.numPages + 1).map((pdfPageNumber) =>
+        this.pdfDocument.getPage(pdfPageNumber).then((pdfPage) =>
+          pdfPage.getViewport(1.0).width
+        )
+      )
+    ).then((pdfPageWidths) => {
+      this.maxPageWidth = Math.max(...pdfPageWidths);
+      this.zoomFit();
+      // console.log('Max width: ' + this.maxPageWidth);
```

Comment:
We can probably remove this `console.log` comment and the similar comments below 😄

Reply:
Done.